lib/native-client.js

```diff
@@ -118,94 +118,6 @@ class NativeClient extends EventEmitter {
   }
 
   /**
-   * Execute a buildInfo command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  buildInfo(callback) {
-    this.database.admin().command({'buildInfo': 1}, {}, (error, result) => {
-      if (error) {
-        return callback(this._translateMessage(error));
-      }
-      callback(null, result);
-    });
-  }
-
-  /**
-   * Execute a hostInfo command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  hostInfo(callback) {
-    this.database.admin().command({'hostInfo': 1}, {}, (error, result) => {
-      if (error) {
-        return callback(this._translateMessage(error));
-      }
-      callback(null, result);
-    });
-  }
-
-  /**
-   * Execute a connectionStatus command on the currently connected instance.
-   *
-   * @param {Function} callback - The callback.
-   */
-  connectionStatus(callback) {
-    // TODO: Don't require admin db...?
-    this.database.admin().command({'connectionStatus': 1}, {}, (error, result) => {
-      if (error) {
-        return callback(this._translateMessage(error));
-      }
-      callback(null, result);
-    });
-  }
-
-  /**
-   * Execute a usersInfo command on the currently connected instance.
-   *
-   * @see https://docs.mongodb.com/manual/reference/command/usersInfo/
-   *
-   * @param {String} authenticationDatabase - The database name.
-   * @param {Object} options - The usersInfo options document, with keys:
-   *  1.  usersInfo - One of:
-   *      - a user name, e.g. 'dba-user', or
-   *      - a list of users and databases, e.g. [
-   *        {user: 'dba-user1', db: 'candy'},
-   *        {user: 'dba-user2', db: 'bar'}
-   *      ], or
-   *      - the special integer 1, meaning list all users
-   *      (can be with showCredentials but not with showPrivileges).
-   *  2.  showCredentials (boolean)
-   *  3.  showPrivileges (boolean)
-   * @param {Function} callback - The callback.
-   * @return {null}
-   */
-  usersInfo(authenticationDatabase, options, callback) {
```

Comment:
This method is publically available in DataService. Will need to verify Compass is not using it and remove it from the public API.